{"uuid":"ccfc3f6a-ee81-489a-95ba-68ad5c7feb82","historyId":"2e79d5b55bba9b56cb9170b0bbb1832a","fullName":"AwaitilityTests.waitUntilAsserted","labels":[{"name":"package","value":"AwaitilityTests"},{"name":"testClass","value":"AwaitilityTests"},{"name":"testMethod","value":"waitUntilAsserted"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"AwaitilityTests"},{"name":"host","value":"DESKTOP-GDSL4V8"},{"name":"thread","value":"11740@DESKTOP-GDSL4V8.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"waitUntilAsserted","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Condition with alias 'My custom message' didn't complete within 10 seconds because assertion condition defined as a Lambda expression in AwaitilityTests 1 expectation failed.\nExpected status code <400> but was <200>.\n.","trace":"org.awaitility.core.ConditionTimeoutException: Condition with alias 'My custom message' didn't complete within 10 seconds because assertion condition defined as a Lambda expression in AwaitilityTests 1 expectation failed.\nExpected status code <400> but was <200>.\n.\r\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\r\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\r\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\r\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\r\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\r\n\tat AwaitilityTests.waitUntilAsserted(AwaitilityTests.java:19)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\r\nCaused by: java.util.concurrent.TimeoutException\r\n\tat java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)\r\n\tat org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)\r\n\tat org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)\r\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)\r\n\t... 36 more\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Request","source":"e94e0c33-b1a7-430f-8123-2b4e7c9ca55b-attachment.html","type":"text/html"},{"name":"HTTP/1.1 200 OK","source":"888eca8e-787f-4de3-bf57-efef716b6a72-attachment.html","type":"text/html"}],"parameters":[],"start":1750253120886,"stop":1750253131068}